tests/end_to_end/test_basic_cases/test_brute_force/test_brute_force_ssh/data/playbooks/generate_events.yaml

```diff
@@ -33,14 +33,23 @@
       failed_when:
         - "'Permission denied' not in result.stdout"
 
-    - name: Wait for alert
-      wait_for:
-        timeout: 5
-
 - name: Get alerts file
   hosts: managers
+  vars:
+    ts:
```

Comment:
```suggestion
    timestamp:
```

Reply:
Is necessary to define this null var at this level?

Reply:
No, deleted in 4cebf5d